tests/periph/eeprom/Makefile

```diff
@@ -4,4 +4,8 @@ FEATURES_REQUIRED += periph_eeprom
 
 USEMODULE += shell_cmds_default  # provides reboot command
 
+ifeq (native, $(BOARD))
```

Comment:
The problem seems to be rather the `--eeprom` parameter not being passed properly to pyterm.

I propose to use
```diff
diff --git a/boards/native/Makefile.include b/boards/native/Makefile.include
index 0e7016caed..d886feb136 100644
--- a/boards/native/Makefile.include
+++ b/boards/native/Makefile.include
@@ -113,8 +113,12 @@ EEPROM_FILE ?= $(BINDIR)/native.eeprom
 
 # set the eeprom file flags only when the periph_eeprom feature is used.
 ifneq (,$(filter periph_eeprom,$(FEATURES_USED)))
-  EEPROM_FILE_FLAGS = --eeprom $(EEPROM_FILE)
-  TERMFLAGS += $(EEPROM_FILE_FLAGS)
+  ifeq (native,$(RIOT_TERMINAL))
+    EEPROM_FILE_FLAGS = --eeprom $(EEPROM_FILE)
+  else
+    EEPROM_FILE_FLAGS = '\-\-eeprom $(EEPROM_FILE)'
+  endif
+  TERMFLAGS += --process-args $(EEPROM_FILE_FLAGS)
 endif
 
 VCAN_IFNUM ?= 0